### PR TITLE
doc(quick install): streamline documentation

### PR DESCRIPTION
**Pull request by jayaddison**
General principles followed while creating these changes:

- Simplify language and length of sentences where possible, while retaining the same meaning
- Replace lengthy related documentation with shorter commands where possible
- Recommend easier approaches (for example, hosted Jitsi) to reduce user effort
- Arrange the document so that users can follow it in-order and get confirmation of a working system early
- Commit each documentation change individually to allow easy reviews & reverts

I've tested the process locally using Ubuntu 19.10.  It'd be good to confirm that `/etc/issue` and `apt-add-repository` exist as far back as Debian Jessie if this is still to be indicated as the oldest supported version.

I haven't modified any of the 'advanced' or 'uninstall' sections.

### CODE REVIEW

File: doc/quick-install.md [hunk 1]
```diff
@@ -1,16 +1,25 @@
...
+1. Ubuntu 14.04 or later
 
-b.) You only need to do this if you want to ___host your own Jitsi server___. If you just want to have a video conference with someone, use https://meet.jit.si instead.
+To check the operating system available, you can run `cat /etc/issue` at the command-line.
+
+_Note_: All commands require superuser privileges. If you are logged in as a regular user, you may need to prepend `sudo` to each of the commands.
+
+### Platform-specific notes
+
+On Debian Wheezy, [libc needs to be updated](http://lists.jitsi.org/pipermail/users/2015-September/010064.html).
```
Comment (jayaddison) [inline doc/quick-install.md]: This link (and the web server `lists.jitsi.org`) doesn't appear to be working.  Although it's not ideal, the solution which maintains the content most accurately may be to replace this with an Internet Archive link.

I considered pulling the contents of the archived post into the document here, but given that it's for a relatively old version of Debian (June, 2016), it is probably preferable to keep the quick install document short.

File: doc/quick-install.md [hunk 2]
```diff
@@ -1,64 +1,89 @@
...
 
-Also note that a recent default Ubuntu installation has only the `main` repository enabled, and Jitsi Meet needs packages from `universe`. Check your `/etc/apt/sources.list` file, and if `universe` is not present refer to [Ubuntu's documentation](https://help.ubuntu.com/community/Repositories/Ubuntu) on how to enable it. (Usually it amounts to copying the `main` lines and changing to `universe`.)
+The installation steps documented here are intended for Debian-based GNU/Linux systems.
 
-N.B.:
+Systems running the following versions are supported by this guide:
 
-a.) All commands are supposed to be run by root. If you are logged in as a regular user with sudo rights, please prepend ___sudo___ to each of the commands.
+1. Debian 8 (Jessie) or later
+1. Ubuntu 14.04 or later
```
Comment (damencho) [inline doc/quick-install.md]: I have a question about these versions, I'm not sure about those. Is that something you tested or we have it stated somewhere?

Comment (jayaddison) [reply doc/quick-install.md]: I've carried these over from the existing version in the quick-install guide ([diff ref](https://github.com/jitsi/jitsi-meet/pull/5227/files#diff-39d47d1432f7af0aa02f3e913097320aL3)), but no, I can't confirm testing of the instructions going back to those OS versions.

Comment (jayaddison) [reply doc/quick-install.md]: Another small detail to add - during uninstall (not currently modified in this changeset), there were some slight issues around package name mismatches.

I'd considered trying to clean that section up, but due to concerns about exactly this kind of backwards-compatibility testing, I didn't feel confident in applying changes which only worked locally (Ubuntu 19.10).

File: doc/quick-install.md [hunk 3]
```diff
@@ -1,64 +1,89 @@
...
 
-Simply run the following in your shell
+You'll need to share this hostname or IP address with conference participants who will use your service.
 
-```sh
-/usr/share/jitsi-meet/scripts/install-letsencrypt-cert.sh
-```
+### Confirm that your installation is working
+
+Launch a web browser (Chrome, Chromium or latest Opera) and enter the hostname or IP address from the previous step into the address bar.
```
Comment (jayaddison) [inline doc/quick-install.md]: NB: From reading other issues, it sounds like Safari support including video may be enabled soon.  At that point it might be nice to update this to list the three highest browsers based on [market share](https://en.wikipedia.org/wiki/Usage_share_of_web_browsers#Summary_tables) which are (in order, at the time of writing): Chrome, Safari, Firefox

Comment (damencho) [reply doc/quick-install.md]: Once that lands we need to update ... there is a notification to be removed, unsupported browser detection ... many things, but lets land it and we will be doing that.

File: doc/quick-install.md [hunk 4]
```diff
@@ -1,14 +1,12 @@
 # Jitsi Meet quick install
 
-This document describes the required steps for a quick Jitsi Meet installation on a Debian based GNU/Linux system. Debian 8 (Jessie) or later, and Ubuntu 14.04 or later are supported out-of-the-box.
-
-Also note that a recent default Ubuntu installation has only the `main` repository enabled, and Jitsi Meet needs packages from `universe`. Check your `/etc/apt/sources.list` file, and if `universe` is not present refer to [Ubuntu's documentation](https://help.ubuntu.com/community/Repositories/Ubuntu) on how to enable it. (Usually it amounts to copying the `main` lines and changing to `universe`.)
+This guide helps you  ___host your own Jitsi server___. If you want to have a video conference without setting up any infrastructure, use https://meet.jit.si instead.
 
-N.B.:
+This document describes the required steps for a quick Jitsi Meet installation on a Debian based GNU/Linux system. Debian 8 (Jessie) or later, and Ubuntu 14.04 or later are supported out-of-the-box.
```
Comment (damencho) [inline doc/quick-install.md]: These I think are not relevant anymore. I would say Ubuntu 18.04 , and Debian 9.

File: doc/quick-install.md [hunk 5]
```diff
@@ -22,50 +20,43 @@ Then add the same FQDN in the `/etc/hosts` file, associating it with the loopbac
...
 
-### Add the repository
+### Add the Jitsi package repository
 ```sh
 echo 'deb https://download.jitsi.org stable/' >> /etc/apt/sources.list.d/jitsi-stable.list
 wget -qO -  https://download.jitsi.org/jitsi-key.gpg.key | apt-key add -
 ```
 
-### Update the package lists
+### Generate a Let's Encrypt certificate (optional, recommended)
```
Comment (damencho) [inline doc/quick-install.md]: Why did you move this up in the list, it should be after installing jitsi-meet?

File: doc/quick-install.md (hunk 4 above)
Comment (damencho) [reply doc/quick-install.md]: 14.04 for sure is not working because of the systemd requirement. Debian 8 is end of life.

File: doc/quick-install.md (hunk 5 above)
Comment (damencho) [reply doc/quick-install.md]: The installer says to the user to run let's encrypt after installing jitsi-meet.

File: doc/quick-install.md (hunk 4 above)
Comment (jayaddison) [reply doc/quick-install.md]: Are you sure Debian 8 has reached EOL?  It remains listed as LTS on this page: https://wiki.debian.org/LTS

+1 on switching Ubuntu to 18.04 (current LTS)

File: doc/quick-install.md (hunk 5 above)
Comment (jayaddison) [reply doc/quick-install.md]: My idea here was that we could have users prepare their Let's Encrypt certificate in advance, and then they'd be able to select it directly during the installer, without having to 'go back around' and waste a few cycles on self-signed certificate generation.

(Ok, that's slightly tongue-in-cheek about the wasted CPU cycles; but if we can save a few steps then I figure we should)

The installer text could remain as-is for anyone following separate install guidelines, accidentally misses this step, or changes their mind and decides to use Let's Encrypt after starting the installer.

File: doc/quick-install.md (hunk 4 above)
Comment (damencho) [reply doc/quick-install.md]: Well it ends in few months :) June 30, 2020 so I would say let's remove it since we are here

File: doc/quick-install.md (hunk 5 above)
Comment (damencho) [reply doc/quick-install.md]: That will not work. That script asks just for email and grabs the rest of the information from the installed jitsi-meet. That was the idea of that automation. Otherwise, we can just remove the script and just add a note, "hey do the let's encrypt thing" and tie a link to their howto.
Maybe when you are there add a note here that mobile will not work without this step! I think that is also important.

File: doc/quick-install.md (hunk 4 above)
Comment (jayaddison) [reply doc/quick-install.md]: That seems reasonable; https://wiki.debian.org/DebianReleases was useful to confirm that Debian 9 EOL in June 2020 should meet their LTS target of 5 years for stable releases.

File: doc/quick-install.md (hunk 5 above)
Comment (jayaddison) [reply doc/quick-install.md]: Understood; I'll relocate the Let's Encrypt instructions to their original position, and then add that notice regarding mobile clients.

File: doc/quick-install.md [hunk 6]
```diff
@@ -1,14 +1,12 @@
...
+This document describes the required steps for a quick Jitsi Meet installation on a Debian based GNU/Linux system. Debian 9 (Stretch) or later, and Ubuntu 18.04 (Bionic Beaver) or later are supported out-of-the-box.
 
-N.B.:
+On Ubuntu systems, Jitsi requires dependencies from Ubuntu's `universe` package repository.  To ensure this is enabled, run `apt-add-repository universe` at the command-line.
 
-a.) All commands are supposed to be run by root. If you are logged in as a regular user with sudo rights, please prepend ___sudo___ to each of the commands.
-
-b.) You only need to do this if you want to ___host your own Jitsi server___. If you just want to have a video conference with someone, use https://meet.jit.si instead.
+_Note_: All commands require superuser privileges. If you are logged in as a regular user, you may need to prepend `sudo` to each of the commands.
```
Comment (jayaddison) [inline doc/quick-install.md]: :warning: This may not work for some commands - https://github.com/jitsi/jitsi-meet/pull/5227/files#diff-39d47d1432f7af0aa02f3e913097320aR25 for example.  Let me see if I can resolve this, or alter the recommendation to use `sudo` if not.

**Comment by saghul** [comment]
Thanks a ton @jayaddison ! ❤️
